src/main/java/com/openelements/conduct/integration/slack/SlackConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.openelements.conduct.integration.slack;
+
+import com.openelements.conduct.data.ResultHandler;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(
+        name = "conductIntelligence.integration.slack.enabled",
+        havingValue = "true",
+        matchIfMissing = false
+)
+public class SlackConfig {
+
+    @Value("${conductIntelligence.integration.slack.token}")
+    private String slackToken;
+
+    @Value("${conductIntelligence.integration.slack.channelId}")
```

Comment:
Please check the current main version. The config param name pattern has changed.
```suggestion
    @Value("${guardian.integration.slack.channelId}")
```
You need to fix that at several places